src/app/components/chat-window.tsx

```
@@ -103,10 +103,40 @@ export const ChatWindow = ({
+      }
+
+      // Ctrl/Cmd + / or Ctrl/Cmd + K to focus input
+      if ((e.metaKey || e.ctrlKey) && (e.key === "/" || e.key === "k")) {
+        e.preventDefault();
+        textareaRef.current?.focus();
+      }
+    };
+
+    window.addEventListener("keydown", handleKeyDown);
+    return () => window.removeEventListener("keydown", handleKeyDown);
+  }, [isSubmitted, isStreaming, stop]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)